# height in meters
intake: 0.0
L1: 0.43 # doesn't work
L2: 0.75
L3: 1.16
L4: 1.73 # high side = 1.75
stow: 0.25 

avoid_elevator_switch: elevator_avoid_limit_switch
outtake_switch: roller_limit_switch

tolerance: 0.03 # m

controller_name: elevator_controller
joint_name: elevator_leader